projects/1.16/assets/meet-your-fight/meetyourfight/lang/zh_cn.json

```
@@ -1 +1,86 @@
-{}
\ No newline at end of file
+{
+	"item.meetyourfight.haunted_bell": "被诅咒的铃铛",
+	"item.meetyourfight.haunted_bell.desc": "召唤敲钟人",
+	"item.meetyourfight.phantoplasm": "幽魂质",
+	"item.meetyourfight.passages_toll": "旅程之钟",
+	"item.meetyourfight.passages_toll.desc": "将你传送至最多16格远的任意方块上",
+	"item.meetyourfight.spectres_eye": "幽灵之眼",
+	"item.meetyourfight.spectres_eye.desc": "使附近的敌对生物发光",
+	"item.meetyourfight.spectres_grasp": "幽灵之握",
+	"item.meetyourfight.phantasmal_rifle": "幻影步枪",
+	"item.meetyourfight.phantasmal_rifle.desc": "穿过方块抛射",
+	"item.meetyourfight.aether_glazed_cupcake": "以太盖蛋糕",
+	
+	"item.meetyourfight.devils_ante": "恶魔赌注",
+	"item.meetyourfight.devils_ante.desc": "召唤福尔图娜女爵",
+	"item.meetyourfight.fortunes_favor": "命运之眷",
+	"item.meetyourfight.slicers_dice": "切割者之骰",
+	"item.meetyourfight.slicers_dice.desc": "有几率造成更多伤害",
+	"item.meetyourfight.ace_of_iron": "铁之王牌",
+	"item.meetyourfight.ace_of_iron.desc": "有几率无视攻击",
+	"item.meetyourfight.cocktail_cutlass": "甜心鸡尾酒之刃",
+	"item.meetyourfight.cocktail_cutlass.desc": "攻击有几率给予药水效果",
+	"item.meetyourfight.cocktail_shotgun": "猎人之射",
+	"item.meetyourfight.cocktail_shotgun.desc": "有几率射出负面的喷溅型药水",
+	"item.meetyourfight.velvet_fortune": "幸运天鹅绒",
+	
+	"item.meetyourfight.desc.luck": "会被运气影响",
+	
+	"item.meetyourfight.fossil_bait": "化石之饵",
+	"item.meetyourfight.fossil_bait.desc": "召唤沼泽巨颌怪",
+	"item.meetyourfight.mossy_tooth": "藓牙",
+	"item.meetyourfight.bone_raker": "骨质钉耙",
+	"item.meetyourfight.depth_star": "深度之星",
+	"item.meetyourfight.depth_star.desc": "可以造成短程冲击波",
+	"item.meetyourfight.caged_heart": "囚笼之心",
+	"item.meetyourfight.caged_heart.desc": "降低超过最大生命值的25%的伤害",
+	"item.meetyourfight.marshy_delight": "湿软泥泞的愉悦美食",
+	
+	"item.meetyourfight.music_disc_magnum": "音乐唱片",
+	"item.meetyourfight.music_disc_magnum.desc": "Monkey Warhol - Magnum",
+	
+	"item.meetyourfight.bellringer_spawn_egg": "敲钟人生成蛋",
+	"item.meetyourfight.dame_fortuna_spawn_egg": "福尔图娜女爵生成蛋",
+	"item.meetyourfight.swampjaw_spawn_egg": "沼泽巨颌生成蛋",
+	
+	"entity.meetyourfight.bellringer": "敲钟人",
+	"entity.meetyourfight.dame_fortuna": "福尔图娜女爵",
+	"entity.meetyourfight.swampjaw": "沼泽巨颌怪",
+	
+	"status.meetyourfight.boss_nearby": "太接近别的BOSS了",

```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
	"status.meetyourfight.boss_nearby": "太接近别的Boss了",

```